src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars--1">
+      <div class="stars__star active"></div>
+      <div class="stars__star"></div>
```

Comment:
It's better to use nth-of-type / nth-child pseudo-classes instead of class active in this case.

Reply:
Problem is fixed